modules/server/src/network/queries/index.ts

```diff
@@ -46,22 +45,3 @@ export const gqlAggregationTypeQuery = `#graphql
 		}
 	}
 `;
-
-export const aggregationsQuery = /* GraphQL */ `
-	#graphql
-	{
-		__typename
-		bucket_count
-		buckets {
-			key
-			doc_count
-		}
-	}
-`;
-
-/**
- * Returns gql query for gql type
- */
-export const supportedAggregationQueries = new Map<SupportedAggregation, string>();
-supportedAggregationQueries.set(SUPPORTED_AGGREGATIONS.Aggregations, aggregationsQuery);
-supportedAggregationQueries.set(SUPPORTED_AGGREGATIONS.NumericAggregations, '');
```

Comment:
forwarding relevant part of query to network nodes, instead of splicing in this static query for each field
resulting queries are smaller and only return requested fields (half the point of GQL)